ocs_ci/ocs/utils.py

```diff
@@ -1681,3 +1681,17 @@ def collect_pod_container_rpm_package(dir_name):
                     go_log_file_name = f"{package_log_dir_path}/{pod_obj.name}-{container_name}-go-version.log"
                     with open(go_log_file_name, "w") as f:
                         f.write(go_output)
+
+
+def is_dr_scenario():
+    """
+    Check if it is RDR or MDR setup
+
+    Returns:
+        bool: return True if it is rdr or mdr setup otherwise False
+
+    """
+    for cluster in ocsci_config.clusters:
```

Comment:
@OdedViner we don't have to iterate over all the clusters, since scenario would be passed as common conf in run-ci, it should be available in all contexts.
```suggestion
    if cluster.MULTICLUSTER['multicluster_mode'] in ("metro-dr", "regional-dr"):
        return True
```

Reply:
done